journal/updated/01-journal.adoc

```diff
@@ -767,251 +767,270 @@ nevertheless, afterward it yields the peaceable fruit of righteousness,
-"`Where two or three are gathered together in my name,
+"`Where two or three are gathered together in My name,
 there am I in the midst of them,`" Matt. 18:20.
 
 So we all agreed to meet together, but none of us had a house of his own to meet in.
 We determined therefore to meet upon a hill in a common,
 as near as we could for the conveniency of each other,
-we living some miles distant one from another.
+we living some miles distant from one another.
 There we met in silence, to the wonder of the country.
 When the rain and weather beat upon us on one side of the hill, we went to the other side.
 We were not free to go into any of the neighbours`' enclosures, for they were so blind,
 dark, and ignorant, that they looked upon us as witches, and would go away from us,
 some crossing themselves with their hands about their foreheads and faces.
 
 Thus we continued for some time, till two of them left me;
-one of them was put apprentice by Friends to William Shewen, of London.
-The other young man was a soldier in Oliver Cromwell`'s days, and he got,
-as soon as he could, to be disbanded, so he went to Gloucestershire,
+one of them was bound as an apprentice by Friends to William Shewen, of London.
+The other young man was a soldier in Oliver Cromwell`'s days, and
+as soon as he could, he got dismissed from service, and so went to Gloucestershire
 and lived among Friends.
-The third was one William Davies, that lived hard by me,
+The third was one William Davies, who lived close to me,
 and we met together for some time; but one time he stayed longer than usual,
-and a foul dark spirit possessed him, so that the little time we were together,
-was not comfortable to me; and when we had broke up the meeting, by way of discourse,
-he asked me, How I did think we should stand in the face of the whole country.
+and a foul and dark spirit possessed him, so that the little time we were together
+was not comfortable to me. And when we had broken up the meeting by way of discourse,
+he asked me how I thought we would be able to stand in the face of the whole country.
 I answered him with great zeal, "`The Serpent, the Serpent,
 the same that beguiled the woman in Paradise, has beguiled you;
 you will not be able to stand.`"
-And while we were yet discoursing, I saw my master coming, who was also convinced,
-but was not faithful to that good spirit that convinced him of the truth,
-and showed him what he ought to do, but did it not; yet he continued loving to Friends,
-and frequented their meetings to his latter end.
-As I looked back, I saw him coming towards us, with two women following him,
-the one was his wife, my cruel mistress, the other was his sister;
-they both had staves in their hands, and when they came unto us,
-the sister began to beat her brother,
-my master and my mistress set a beating of William Davies.
+And while we were yet discoursing, I saw my master coming,
+who later was also convinced,
```

Comment:
we need to take out the word "later", because he was already convinced, but just wasn't being faithful to the Spirit that convinced him.